File: 프로그래머스/0/181925. 수 조작하기 2/수 조작하기 2.js
function solution(numLog) {
    let result = numLog[0];
    numLog.shift()
    return numLog.map( i => {
        if(result + 1 === i) {
            result =i
            return 'w'
        }
        if(result - 1 === i) {
            result =i
            return 's'
        }
        if(result + 10 === i) {
            result =i
            return 'd'
        }
        if(result - 10 === i) {
            result =i
            return 'a'
        }
    }).join('')
}

File: 프로그래머스/0/181925. 수 조작하기 2/수 조작하기 2.py
def solution(numLog):
    result = ''
    for i,j in enumerate(numLog[1:]):
        if numLog[i+1] - numLog[i] == 1:
            result += 'w'
        elif numLog[i+1] - numLog[i] == -1:
            result += 's'
        elif numLog[i+1] - numLog[i] == 10:
            result += 'd'
        else:
            result += 'a'
    return result